services/movie_session.py

```diff
@@ -0,0 +1,53 @@
+from datetime import datetime
+from django.db.models import QuerySet
+from db.models import MovieSession, Movie, CinemaHall
+
+
+def create_movie_session(
+        movie_show_time: datetime,
+        movie_id: int,
+        cinema_hall_id: int
+) -> None:
```

Comment:
You may also change functions `create_movie_session()`, `update_movie_session()`, and `delete_movie_session()` so that they return created/modified/deleted objects as well.